File: dev-docs/snippets/documents/revoke-group-permission.swift
import JsBaoClient

// Revoke a group's permission on a document. Swift returns an untyped
// `[String: Any]` (vs JS `{ success }`).
func revokeGroupPermission(client: JsBaoClient, documentId: String) async throws {
  // #region example
  let result = try await client.documents.revokeGroupPermission(
    documentId: documentId, groupType: "team", groupId: "eng"
  )
  // #endregion example
  _ = result
}

File: dev-docs/snippets/documents/revoke-group-permission.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Revoke a group's permission on a document.
export async function revokeGroupPermission(
  client: JsBaoClient,
  documentId: string,
) {
  // #region example
  const { success } = await client.documents.revokeGroupPermission(
    documentId,
    "team",
    "eng",
  );
  // #endregion example
  return success;
}
